From patchwork Mon Jan 13 09:56:02 2014
Content-Type: text/plain; charset="utf-8"
MIME-Version: 1.0
Content-Transfer-Encoding: 7bit
Subject: [RESEND-2] updated automake files for libjson-c (LP: #1268005)
From: Colin King <colin.king@canonical.com>
X-Patchwork-Id: 309769
Message-Id: <1389606962-12027-1-git-send-email-colin.king@canonical.com>
To: fwts-devel@lists.ubuntu.com
Date: Mon, 13 Jan 2014 09:56:02 +0000

From: Colin Ian King <colin.king@canonical.com>

This enables builds with json-c 0.11.  Thanks to Jeffrey Bastian for this
fix.

Signed-off-by: Jeffrey Bastian <jbastian@redhat.com>
Signed-off-by: Colin Ian King <colin.king@canonical.com>
Acked-by: Alex Hung <alex.hung@canonical.com>

From http://patchwork.ozlabs.org/patch/309769/mbox/
Upstream-status: Submitted

---
src/Makefile.am           | 2 +-
 src/lib/src/Makefile.am   | 2 +-
 src/utilities/Makefile.am | 2 +-
 3 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/src/Makefile.am b/src/Makefile.am
index df685d5..d65b066 100644
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -87,7 +87,7 @@ fwts_SOURCES = main.c 				\
 	uefi/securebootcert/securebootcert.c	\
 	uefi/uefivarinfo/uefivarinfo.c
 
-fwts_LDFLAGS = -ljson -lm
+fwts_LDFLAGS = -ljson -ljson-c -lm
 
 fwts_LDADD = \
 	-Llib/src -lfwts \
diff --git a/src/lib/src/Makefile.am b/src/lib/src/Makefile.am
index d708f34..a07d5e2 100644
--- a/src/lib/src/Makefile.am
+++ b/src/lib/src/Makefile.am
@@ -7,7 +7,7 @@ AM_CPPFLAGS = \
 pkglib_LTLIBRARIES = libfwts.la
 
 libfwts_la_LDFLAGS = 			\
-	-lm -lpcre -lpthread -ljson 	\
+	-lm -lpcre -lpthread -ljson -ljson-c 	\
 	-version-info 1:0:0 		\
 	-L$(top_srcdir)/src/acpica/source/compiler \
 	-lfwtsiasl
diff --git a/src/utilities/Makefile.am b/src/utilities/Makefile.am
index 427c44f..cc2e115 100644
--- a/src/utilities/Makefile.am
+++ b/src/utilities/Makefile.am
@@ -2,5 +2,5 @@ AM_CPPFLAGS = -Wall -Werror -Wextra
 
 bin_PROGRAMS = kernelscan
 kernelscan_SOURCES = kernelscan.c
-kernelscan_LDFLAGS = -ljson -lpcre
+kernelscan_LDFLAGS = -ljson -ljson-c -lpcre
 
